src/modules/UI/scenes/SendConfirmation/SendConfirmation.ui.js

```diff
@@ -190,52 +190,57 @@ export class SendConfirmation extends Component<Props, State> {
                 isEditable={this.props.isEditable}
               />
 
-              <View style={[styles.feeArea]}>
-                {(!!this.props.networkFee || !!this.props.parentNetworkFee) && <Text style={[styles.feeAreaText]}>{this.networkFeeSyntax()}</Text>}
-              </View>
-
-              {!!destination && (
-                <Recipient style={{ paddingHorizontal: 20 }}>
-                  <Recipient.Text>
-                    <Text>{SEND_TO_DESTINATION_TEXT}</Text>
-                  </Recipient.Text>
-                </Recipient>
-              )}
-
-              {!!this.props.uniqueIdentifier && (
-                <UniqueIdentifier>
-                  <UniqueIdentifier.Text>
-                    <Text>{uniqueIdentifierText(this.props.currencyCode, this.props.uniqueIdentifier)}</Text>
-                  </UniqueIdentifier.Text>
-                </UniqueIdentifier>
-              )}
-
-              {authRequired === 'pin' && (
-                <Scene.Row style={styles.row}>
-                  <Text style={styles.rowText}>{s.strings.four_digit_pin}</Text>
-
-                  <View style={styles.pinInputSpacer} />
-
-                  <View style={styles.pinInputContainer}>
-                    <PinInput ref={ref => (this.pinInput = ref)} fontSize={20} onChangePin={this.handleChangePin} returnKeyType="done" />
-                  </View>
-                </Scene.Row>
-              )}
+              <Scene.Padding style={{ paddingHorizontal: 54 }}>
+                <Scene.Item style={{ alignItems: 'center' }}>
+                  {(!!this.props.networkFee || !!this.props.parentNetworkFee) && (
+                    <Scene.Row style={{ paddingVertical: 4 }}>
+                      <Text style={[styles.feeAreaText]}>{this.networkFeeSyntax()}</Text>
+                    </Scene.Row>
+                  )}
+
+                  {!!destination && (
+                    <Scene.Row style={{ paddingVertical: 4 }}>
+                      <Recipient.Text style={{}}>
+                        <Text>{SEND_TO_DESTINATION_TEXT}</Text>
+                      </Recipient.Text>
+                    </Scene.Row>
+                  )}
+
+                  {!!this.props.uniqueIdentifier && (
+                    <Scene.Row>
+                      <UniqueIdentifier.Text>
+                        <Text>{uniqueIdentifierText(this.props.currencyCode, this.props.uniqueIdentifier)}</Text>
+                      </UniqueIdentifier.Text>
+                    </Scene.Row>
+                  )}
+
+                  {authRequired === 'pin' && (
+                    <Scene.Row style={{ width: '100%', justifyContent: 'flex-start' }}>
```

Comment:
I personally don't mind two style properties being written in-line but it begs the question of how much is too much for inline styles.